docs/CONTRIBUTING.md

```diff
@@ -7,19 +7,23 @@ title: Contributing
 
 ## Who can contribute?
 
-Anyone who has a GitHub account can contribute to the documentation content.
+Anyone who has a GitHub account can contribute to the documentation content:
+
+- Users within the Scaleway organization can open pull requests directly in the docs-content repository.
+
+- External users (i.e. not part of the Scaleway organization) must fork the repository and open a pull request from the forked repository.
```

Comment:
```suggestion
- External users (i.e. not part of the Scaleway organization) **must fork the repository and open a pull request from the forked repository**.
```

Reply:
Can we link the specific procedure for internal users in the Scaleway orga and the one for users not in the orga here? So people don't have to scroll through the first procedure if they can't use it